### ISSUE TITLE
improve list of Gitter rooms on front page

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-lang#688]
currently:

<img width="792" alt="screen shot 2017-07-13 at 10 26 40 am" src="https://user-images.githubusercontent.com/161079/28171244-c7937934-67b5-11e7-944c-6f6cd9aa0968.png">

the first three are good choices, but then it goes south. @jducoeur suggested including the scala-js room, for example

**Comment by heathermiller** [comment]
Please make a list and I can add them. These are merely placeholders.

**Comment by jducoeur** [comment]
A few opinions:

I think scala-js/scala-js is totally appropriate here: it's a very large and open user-centric group, increasingly central to the ecosystem.  It's a good group for people to hit early on.

Possibly also scala-native, but that's still in-development enough that I don't know whether it's appropriate yet.  (I'd probably recommend asking the folks there whether they want to be listed.)

Also possibly scala/job-board, which seems to be developing steady if modest traffic -- helps underline that there are jobs in this technology.

The pickling and scala-xml groups are low-volume and extremely specialized; I don't think they're ideal for here, because they look a little moribund if you just drop in.  (And the link for scala-parser-combinator is broken.)

scala-india is interesting, but so low-volume that, again, I sort of think it doesn't put our best foot forward.

I'm tempted to suggest typelevel/cats, which is an increasingly significant part of the ecosystem, and is a lively and relatively friendly discussion.  Your call on whether that makes sense for this page or not.

**Comment by SethTisue** [comment]
There is already a list at http://www.scala-lang.org/community/, in the Chat Rooms section: scala/scala, scala/center, scala/contributors, scala/moocs, spark-scala/Lobby, scala/job-board.

I'd suggest using this same list, but adding scala-js, since it's nearly 1.0 now.

Heather, you would know best if scala/moocs should be included or not.

I think it's better to keep the list on the site front page short. Pretty much only newbies will use the list — it's better not to overwhelm them. (So, no scala-native or dotty.)

And for the same reason I _don't_ suggest we link to specific rooms on e.g. Cats or Akka...

...but I _do_ suggest we make an exception for Spark, because the volume of beginner questions involving Spark is _really_ high, as a glance at Stack Overflow will indicate.

To summarize, my proposed list: scala/scala, scala/job-board, scala/contributors, scala/center, spark-scala/Lobby, scala-js/scala-js. and possibly scala/moocs.

**Comment by jducoeur** [comment]
I didn't realize that scala/moocs existed.  On the one hand, it's a bit low-volume, but it's *specifically* likely to be interesting to newcomers, so I'd probably argue that it's appropriate for the front page.

And agreed on Spark: it seems to be the number-one vector for people to find Scala these days, so it's worth putting it front-and-center.

**Comment by milessabin** [comment]
I'd like to see typelevel/cats on the front page (and the community list too). It's a good way into an important part of the overall ecosystem. Much more relevant to _general_ users than scala/contributors (which isn't to say that the latter isn't also very important).

**Comment by heathermiller** [comment]
I have no issue with typelevel/cats being on the frontpage and in the community list. 

@milessabin, could you PR it to the community page? That's definitely not contentious. For the front page though, I suppose we should wait for some kind of consensus from others (@SethTisue? Don't know who else?) just to ensure that everyone's OK with this.

**Comment by milessabin** [comment]
@heathermiller will do ...

**Comment by milessabin** [comment]
Done: https://github.com/scala/scala-lang/pull/696.

The header is driven by `_data/chats-forums.yml`, which I guess is what's being discussed in this ticket?

**Comment by heathermiller** [comment]
Thanks @milessabin! And yep, the content of what's in that list is the topic of discussion indeed.

**Comment by SethTisue** [comment]
well, it's a judgment call, but I have already said "no Cats, no Akka, no Play" on the front page and that's still how I feel about it. (I wouldn't mind if scala/contributors didn't make the cut, even.)

if we want to encourage people to discover additional rooms beyond the first handful, a link to that section of the community page, where there's more space to list lots of rooms, would be appropriate.

**Comment by heathermiller** [comment]
What if we separated the list into two sections?

(1) General Scala, and (2) Core community libs/frameworks?

I understand your goal; scala-lang should be focused on the general/core Scala gitter channels, but Akka, Spark, Play, and Typelevel are pretty important community gitter channels. It might help people arriving at the page to know that there are very active, more focused gitter channels they can join if they're interested in FP, concurrency, or web apps.

**Comment by SethTisue** [comment]
sure

**Comment by heathermiller** [comment]
👍 Ok, I'll do this then. Thanks. And I'll add typelevel/cats to it :)

(We can add other major community gitter channels later on as well.)

**Comment by SethTisue** [comment]
this came up again at #798

**Comment by SethTisue** [comment]
I'm working on a PR on this right now.
